apps/admin-web/src/app/app.tsx

```diff
@@ -116,26 +119,14 @@ export const App = () => {
           fullWidth
           id="outlined-search"
           type="search"
-          label="Filter"
+          label="Search"
           variant="outlined"
-          onChange={filter}
+          onChange={debounce(sort, 200)}
         />
       </Container>
     );
   };
 
-  const _renderTypography = () => {
-    const hasBeenFiltered = filterValue !== '';
-    const displayText = hasBeenFiltered
-      ? 'No Results Found. Please confirm the value that has been entered is correct.'
-      : 'No Routes Available.';
-    return (
-      <Typography align="center" sx={{ mt: 10 }}>
-        {displayText}
-      </Typography>
-    );
-  };
```

Comment:
No need for this since all routes will still show

Reply:
So search is just a sort?